app/models/ssh_execution_provider.rb

```diff
@@ -29,6 +28,12 @@ def proxy_operation_name
       'ssh'
     end
 
+    def secrets(host)
+      { :ssh_password => ssh_password(host),
+        :key_passphrase => ssh_key_passphrase(host),
+        :sudo_password => sudo_password(host) }
+    end
```

Comment:
Nitpick: Odd formatting.

Reply:
Should be better now

Reply:
I see you prefer the original formatting, feel free to keep it.